### DISCUSSION TITLE
select two label

### DISCUSSION THREAD

**Discussion by TranquilleXav** [EdgeTX/edgetx#3042] [Q&A]
Hello,

Is it possible to make it so that when I select a label, the previously selected label is deselected?
Currently, when I select a label and another one is selected, no model is displayed

It's just a suggestion
thank you for the job!
have a good day

**Comment by MicheleVilla** [comment]
a model could belong to two categories? it could be done that when I want to select two labels I get it with a long pressure, otherwise I select one by deactivating the other

**Comment by pfeerick** [comment]
One way around this is to use the PAGE hardware keys if you just want to change labels, rather than select a second label to filter the model list by. Otherwise, the current expectation is that if you just wanted to change from "heli" to "plane" is that you would have to select "plane" and then deselect "heli". Making it so you can also just change labels was discussed though, just the right way to implement this with the current UI controls was problematic for the initial implementation as part of 2.8.

**Comment by TranquilleXav** [comment]
"it could be done that when I want to select two labels I get it with a long pressure" -> Yes, this technique does work effectively
"otherwise I select one by deactivating the other" -> Ok, I will do that too
"just the right way to implement this with the current UI controls was problematic " ok i understand, no problem, Edgetx is already a very good tool
thank you for your answers
